repo-update.R

```diff
@@ -125,7 +125,7 @@ for (pkg in packages) {
   } else {
     tarball_file <- tarball(pkg, new_ver_string)
     tarball_path <- file.path(webr_contrib_src, tarball_file)
-    new_url <- paste0(cran_url, "src/contrib/", tarball_file)
+    new_url <- paste0(cran_url, "/src/contrib/", tarball_file)
```

Comment:
Ok thanks, added gsub earlier to prevent trailing slash.
FYI, I realized that my cran url was set to https://packagemanager.posit.co/cran/__linux__/jammy/latest/ because of Rocker Docker image. So that's not really a "cran url". I wonder if this edge case has to be handled here or if it's out of scope.